internal/service/streaminstance/model_stream_instance_test.go

```diff
@@ -0,0 +1,174 @@
+package streaminstance_test
```

Comment:
Why is this file named model? Is this a new convention?

Reply:
This convention was established when adding unit testing for search_deployment, the model_* file contains functions related conversion of terraform and sdk models. If you have concerns or an alternative that looks better we can definitely adjust.